packages/redux-variants/variants.js

```diff
@@ -520,20 +506,3 @@ const sortedVariants = createSelector(
     return sortedVariants
   }
 )
-
-export const finalFilteredVariants = createSelector(
```

Comment:
Renamed `sortedVariants` to `finalFilteredVariants` to avoid changing all the imports.